alws/crud/errata.py

```diff
@@ -1249,9 +1249,48 @@ async def release_errata_record(record_id: str):
             pulp,
         )
         db_record.release_status = ErrataReleaseStatus.RELEASED
-        db_record.last_release_log = "Succesfully released"
+        release_log = [
+            f"Record {record_id} successfully released at {datetime.datetime.utcnow()}\n"
+        ]
+        pkg_fields = [
+            RpmPackage.name,
+            RpmPackage.epoch,
+            RpmPackage.version,
+            RpmPackage.release,
+            RpmPackage.arch,
+            RpmPackage.is_modular,
+        ]
+        pulp_pkgs = get_rpm_packages_by_ids(
+            pulp_pkg_ids=[get_uuid_from_pulp_href(pkg) for pkg in pulp_packages],
+            pkg_fields=pkg_fields,
+        )
+        is_modular = False
```

Comment:
Instead of indicating whether the package belongs to a module, I'd prefer to use `db_record.module`, which gives us the `name:stream` of the module. Also, you can get rid of the `is_modular` check below.